plugin.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "CustomAlerts",
     "description": "Create custom Alerts to be notified of important changes on your website or app! ",
-    "version": "4.0.4",
+    "version": "5.0.0",
```

Comment:
this should be 4.1.0?

Reply:
Seemed like a major change, I'll do minor bumps instead.

Reply:
👍  We always keep major version the same as with core. I assume no functionality changes anyway and there are no breaking changes etc so a minor version increase is best anyway.